pkgs/by-name/y-/y-cruncher/package.nix

```diff
@@ -0,0 +1,73 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  nix-update-script,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "y-cruncher";
+  version = "0.8.6.9545";
+
+  src = fetchurl {
+    url = "https://github.com/Mysticial/y-cruncher/releases/download/v${finalAttrs.version}/y-cruncher.v${finalAttrs.version}-static.tar.xz";
+    hash = "sha256-R57uTP45CXGi3+dcH0eyxR2ewjZKNxIyGHyh6mh7FUk=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 y-cruncher -t "$out"/lib/y-cruncher/
+    find Binaries/ -name "*~*" -printf '"Binaries/%f"\n' | xargs install -Dm755 -t "$out"/lib/y-cruncher/Binaries/
```

Comment:
Although meant to be used by the main binaries, I believe theses shouldn't be hidden within `lib/Binaries` because it create a really weird output:
```
result
├── bin
│   └── y-cruncher
├── lib
│   └── y-cruncher
│       ├── Binaries
│       │   ├── 05-A64 ~ Kasumi
│       │   ├── 08-NHM ~ Ushio
│       │   ...
```
Why not putting them into `bin`, and apply the following:
```nix
preBuild = ''
  substituteInPlace Source/Launcher/Launch_Linux.ipp \
    --replace-fail 'Environment::get_process_directory() + "Binaries/"' \
      '"${placeholder "$out"}/bin"'
```